flink-kubernetes-operator/src/main/java/org/apache/flink/kubernetes/operator/controller/FlinkSessionJobController.java

```diff
@@ -114,7 +119,7 @@ public UpdateControl<FlinkSessionJob> reconcile(
         } catch (Exception e) {
             throw new ReconciliationException(e);
         }
-
+        metricManager.onUpdate(flinkSessionJob);
```

Comment:
is this call necessary? The onUpdate at the end of reconcile should take care of it I believe